packaging/functions.sh

```diff
@@ -35,7 +34,7 @@ install_assemblies_mono() {
 	cd "${SRC_PATH}" || exit 1
 	msbuild -verbosity:m -nologo -t:Clean
 	rm -rf "${SRC_PATH:?}/bin"
-	msbuild -verbosity:m -nologo -t:Build -restore -p:Configuration=Release -p:TargetPlatform="${TARGETPLATFORM}"
+	msbuild -verbosity:m -nologo -t:Build -restore -p:Configuration="Release - Mono" -p:TargetPlatform="${TARGETPLATFORM}" -p:Mono=true -p:DefineConstants="MONO"
```

Comment:
This `install` lines a few lines below will need to change exe &rarr; dll too

Reply:
Also https://github.com/OpenRA/OpenRA/blob/bleed/packaging/macos/launcher.m#L134 and https://github.com/OpenRA/OpenRA/blob/bleed/packaging/macos/launcher-mono.m#L241

Reply:
https://github.com/OpenRA/OpenRA/blob/bleed/packaging/linux/openra-server.appimage.in https://github.com/OpenRA/OpenRA/blob/bleed/packaging/linux/openra-server.in https://github.com/OpenRA/OpenRA/blob/bleed/packaging/linux/openra-utility.appimage.in https://github.com/OpenRA/OpenRA/blob/bleed/packaging/linux/openra.appimage.in https://github.com/OpenRA/OpenRA/blob/bleed/packaging/linux/openra.in too!